roles/openshift_storage_nfs/meta/main.yml

```diff
@@ -0,0 +1,14 @@
+---
+galaxy_info:
+  author: Andrew Butcher
+  description: OpenShift NFS Server
+  company: Red Hat, Inc.
+  license: Apache License, Version 2.0
+  min_ansible_version: 1.9
+  platforms:
+  - name: EL
+    versions:
+    - 7
+dependencies:
+- { role: os_firewall }
```

Comment:
I can't wait until the firewalld stuff is worked out and we can kill os_firewall with fire!